ui/css/layout/_typography.scss

```diff
@@ -123,8 +114,7 @@ h6,
 
 p {
   @include font-serif();
-  font-size: 1rem;
-  line-height: 1.69rem;
+  @include size-regular();
```

Comment:
Note that this is a slightly different line-height, but the new value matches the Sketch file.